README.md

```diff
@@ -27,7 +27,7 @@ Starknet Foundry, like its [Ethereum counterpart](https://github.com/foundry-rs/
 
 - [Forge](https://github.com/foundry-rs/starknet-foundry/tree/master/crates/forge): Starknet testing
   framework (like Truffle, Hardhat and DappTools but for Starknet).
-- [Cast](https://github.com/foundry-rs/starknet-foundry/tree/master/crates/cast): All-in-one tool for
+- [Cast](https://github.com/foundry-rs/starknet-foundry/tree/master/crates/sncast): All-in-one tool for
```

Comment:
Didn't change the link to `Sncast` because it would look weird/be inconsistent with Forge.

Reply:
Let me know if you think we should change it though.

Reply:
imo we should probably use one naming convention everywhere to avoid cast vs sncast confusion (and the same would go for forge). my understanding is that forge will be changed to snforge as well, in different PR, right? so do you plan to change the cast naming then?

@MaksymilianDemitraszek please take a look and have a say